src/web/mjs/connectors/HiperCool.mjs

```diff
@@ -0,0 +1,69 @@
+import Connector from '../engine/Connector.mjs';
+import Manga from '../engine/Manga.mjs';
+
+export default class HiperCool extends Connector {
+
+    constructor() {
+        super();
+        super.id = 'hipercool';
+        super.label = 'Hiper Cool';
+        this.tags = [ 'manga', 'portuguese', 'hentai' ];
+        this.url = 'https://hiper.cool';
+    }
+
+    async _getMangas() {
+        let page = 1;
+        let morePages = true;
+
+        let request, data;
+
+        let mangas = [];
+
+        while ( morePages ) {
+            try {
+                request = new Request(new URL('/home/' + page, this.url), this.requestOptions);
+                data = await this.fetchDOM(request, '.news-list > li');
+                page++;
+
+                for (let manga of data) {
+                    mangas.push(
+                        {
+                            id: manga.querySelector('a.news-thumb').pathname,
+                            title: manga.querySelector('div.title').innerText.trim().replace(/\s+\d{2}(?: Final)?$/, '')
```

Comment:
May place `trim` at the end in case spaces emerge at the beginning or end after the `replace` was applied

Reply:
Sounds like a good idea.